packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -88,17 +117,35 @@ export const ActivityLibrary = (props: Props) => {
                 <div className='mt-0.5'>TODO: create custom activity</div>
               </div>
             </div>
+          ) : (
+            <div className='mx-auto grid max-w-7xl grid-cols-2 gap-4 p-4 @2xl:grid-cols-3 @4xl:grid-cols-4 @6xl:grid-cols-5'>
```

Comment:
Container queries are useful as we want to know what's the size of the parent and use that as a baseline for column count. Due to various paddings/margins using normal media queries won't work well

Reply:
in general, I agree but what I don't like about _not_ using is `max-w` is the fact it'll look bad on wide screen monitors. and yeah, there are other ways of doing this but if you have truly wide monitor you'd need to move your head from left to right to see the content 😋 

that said, you're right about using grid template columns, it's a better fit here. I need to up my grid skills! thanks for the suggestion, fixed 👍